### ISSUE TITLE
eval_loss is always 'nan'

### ISSUE THREAD

**Issue by Jiuzhouh** [tloen/alpaca-lora#193]
![image](https://user-images.githubusercontent.com/34101235/228151277-a65586e9-0733-4ab5-9489-de7b705c4fac.png)
I use the same training setting of 7B model (show loss every 5 step and use micro-batch-size = 8). But I cannot get the reasonable eval_loss. Sometimes the training loss would also be 0. After 3 epoch, the model cannot generate any useful information (only blank spaces).

**Comment by zhihui-shao** [comment]
same issue, any suggestions? ~ thanks a lot.

**Comment by nkjulia** [comment]
same issure, any suggestion?

**Comment by Dicer-Zz** [comment]
same issure, any suggestion?

**Comment by JianqiaoLu** [comment]
> ![image](https://user-images.githubusercontent.com/34101235/228151277-a65586e9-0733-4ab5-9489-de7b705c4fac.png) I use the same training setting of 7B model (show loss every 5 step and use micro-batch-size = 8). But I cannot get the reasonable eval_loss. Sometimes the training loss would also be 0. After 3 epoch, the model cannot generate any useful information (only blank spaces).

do you solve it ?
